nvme.c

```diff
@@ -6573,14 +6575,18 @@ static int copy_cmd(int argc, char **argv, struct command *cmd, struct plugin *p
 	const char *d_format = "source range entry format";
 
 	_cleanup_nvme_dev_ struct nvme_dev *dev = NULL;
-	uint16_t nr, nb, ns, nrts, natms, nats;
+	uint16_t nr, nb, ns, nrts, natms, nats, nid;
```

Comment:
naming: pluralize, i.e. "nids"?

Could also use `__u16`; that seems to be the preferred integer style.